qa/rpc-tests/wallet_overwintertx.py.hack

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python2
+# Copyright (c) 2018 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.authproxy import JSONRPCException
+from test_framework.util import assert_equal, initialize_chain_clean, \
+    start_nodes, connect_nodes_bi, sync_blocks, sync_mempools, \
+    wait_and_assert_operationid_status
+
+from decimal import Decimal
+
+class WalletOverwinterTxTest (BitcoinTestFramework):
```

Comment:
This file appears to have been included by mistake, judging by the `.hack` suffix.

Reply:
force-pushed fix